src/main/java/teammates/common/util/Const.java

```diff
@@ -386,7 +386,7 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String FEEDBACK_QUESTION_GETLINK =
                 "Get a submission link to this particular question. "
                 + "Useful if you want students to answer individual questions separately or at different points in time.";
-        public static final String FEEDBACK_QUESTION_CANCEL = "Discard your changes";
+        public static final String FEEDBACK_QUESTION_DISCARDCHANGES = "Discard your changes";
```

Comment:
Looks like we are using 'you' more than 'I' and it'll be easier to standardize on 'you'. However, from a UIX perspective, are they both the same or is one better than the other?